lib/plugins/aws/invoke.js

```diff
@@ -70,6 +80,16 @@ class AwsInvoke {
       Payload: Buffer.from(JSON.stringify(this.options.data || {})),
     };
 
+    if (this.options.context) {
+      let clientContext;
+      if (typeof this.options.context !== 'string') {
+        clientContext = JSON.stringify(this.options.context);
+      } else {
+        clientContext = this.options.context;
+      }
+      params.ClientContext = Buffer.from(clientContext).toString('base64');
+    }
```

Comment:
I think we should follow exactly same logic as with data input, so:
1. Attempt to _parse_ if no `--raw` option was passed
2. In all cases `JSON.stringify` when assigning to params